ifrs17/Utils/Queries.ipynb

```diff
@@ -399,6 +400,74 @@
             "metadata": {},
             "execution_count": 0,
             "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Raw and Ifrs Variables"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public static async Task<T[]> LoadVariablesAsync<T>(this IDataSource querySource, Guid partition) where T : BaseDataRecord",
```

Comment:
Shouldn't this be something like IPartitioned?
You can make the PartitionByReportingNodeAndPeriod generic, I think and reuse this code in another context.